Dockerfile

```diff
@@ -30,6 +32,10 @@ RUN apt-get update \
     && add-apt-repository -y ppa:deadsnakes/ppa  \
     && add-apt-repository ppa:linuxuprising/java \
     && apt-get install --yes jq lsof git cmake make perl psmisc curl wget gcc-11 g++-11 python3.10 python3.10-venv python3-pip python3-dev \
+    && wget https://packages.microsoft.com/config/ubuntu/$(lsb_release -rs)/packages-microsoft-prod.deb -O packages-microsoft-prod.deb \
```

Comment:
I think it is better to move this under condition as well. as if docker is being built for geth only no need to install nethermind pre requirements